src/CascadeMinimizer.cc

```diff
@@ -454,6 +460,15 @@ bool CascadeMinimizer::multipleMinimize(const RooArgSet &reallyCleanParameters,
      Mode 2 -- Full scan over the remaining combinations after mode 1
     */
 
+    // If the barlow-beeston minimisation is being used we can disable it temporarily,
+    // saves time if we don't have to call enable/disable on the CMSHistErrorPropagators
+    // repeatedly for no purpose
+    int currentBarlowBeeston = runtimedef::get(std::string("MINIMIZER_analytic"));
+    runtimedef::set("MINIMIZER_analytic", 0);
+    
+    double backupStrategy = ROOT::Math::MinimizerOptions::DefaultStrategy();
+    ROOT::Math::MinimizerOptions::SetDefaultStrategy(0);
```

Comment:
@nucleosynthesis we force the minimizer strategy to 0 here - found that there was not much point using 1 or 2 as the hesse calls within these would fail due to the presence of the other floating parameters on the inactive pdfs that don't change the NLL value